docs/accessibility/guides/local-development.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Feedback during local development
+sidebar_position: 50
+---
+
+# Feedback during local development
+
+Eventually, the Cypress App will support displaying accessibility feedback during local development. For now, the important thing to note is: you do _not_ need to push your changes and wait for your entire build process to complete.
```

Comment:
I’m a bit cautious about saying what the product will eventually have in the docs. We’ve had some of these statements in the past sit around for years as priorities changed.

Reply:
Fair, removed.